hail/python/hail/docs/getting_started.rst

```diff
@@ -9,12 +9,10 @@ Requirements
 
 Regardless of installation method, you will need:
 
-- `Java 8 JDK
-  <http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html>`_
-  Note: it *must* be version eight. Hail does not support Java versions nine,
-  ten, or eleven due to our dependency on Spark.
-- Python 3.6 or later, we recommend `Anaconda's Python 3
-  <https://www.anaconda.com/download/>`_
+- `Java 8 JDK <http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html>`_ (version **8** required)
+    Note: it *must* be Java **8**. Hail does not support versions 9+ due to our dependency on Spark.
+- Python 3 (version **3.6+** required): We recommend `Miniconda Python 3.7 <https://docs.conda.io/en/latest/miniconda.html>`_
+    Mac/Linux Miniconda users: if ``conda activate`` doesn't work, `please read these instructions <https://conda.io/projects/conda/en/latest/user-guide/install/macos.html#install-macos-silent>`_
```

Comment:
we could just recommend people use `source activate`. That shouldn't suffer the same problems.

This message should be down at the conda activate line, if you want to leave it as `conda activate`.

Also, I prefer the previous "Python 3.6 or later" to "Python 3 (version **3.6+** required)"

Reply:
Source activate is deprecated. Will move

Reply:
ugh, the docs apparently changed since then. I no longer see a strong preference note in documentation, so I don't care which we use (conda activate apparently requires a workaround to use in scripts)